pkg/parse/state.go

```diff
@@ -28,36 +28,40 @@ const (
 	maxRetryInterval             = time.Duration(5) * time.Minute
 )
 
-type sourceStatus struct {
+type commonStatus struct {
 	commit     string
 	errs       status.MultiError
 	lastUpdate metav1.Time
 }
 
-func (gs sourceStatus) equal(other sourceStatus) bool {
-	return gs.commit == other.commit && status.DeepEqual(gs.errs, other.errs)
+func (cs commonStatus) Equal(other commonStatus) bool {
+	return cs.commit == other.commit && status.DeepEqual(cs.errs, other.errs)
```

Comment:
Why do we need to convert `equal` to `Equal`? This method is only called in the same package.